source/documentation/getting_started/quick_start.md

```diff
@@ -1,10 +1,59 @@
 ## Quick start guide
 
-Registers in beta are reliable and are ready for use in products and services.
+Registers which are in beta are reliable and ready for use in products and services. Each register has an open, RESTful API which you can use to access the data in the register. 
```

Comment:
Is the "are in beta" helping clarify that it is the same as "ready to use"? Or it's something to remove?

Reply:
It's a holdover from old language I didn't write, so I would look to you to tell me if we should just remove it.

Reply:
In this instance I can't think of a way to restructure this sentence since "beta" is such an important part of it:

> Registers which are in beta are reliable and ready for use in products and services.

So perhaps just delete this sentence?

Reply:
I'm interested in other opinions, @michaelabenyohai @danmitchell-

Reply:
Even if it's true I think it's not really necessary to put it in the API docs; it's the sort of information that really belongs in more of a frontend context IMO. I've removed the sentence in the latest commit but depending on further feedback I can revisit/reintroduce it.